Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM registry.centos.org/centos/centos:8
 
 MAINTAINER Slavek Kabrda <slavek@redhat.com>
 
-RUN yum install -y https://download.postgresql.org/pub/repos/yum/10/redhat/rhel-8-x86_64/pgdg-redhat-repo-latest.noarch.rpm &&\
+RUN yum install -y https://download.postgresql.org/pub/repos/yum/reporpms/EL-6-x86_64/pgdg-redhat-repo-latest.noarch.rpm &&\
```

Comment:
https://github.com/fabric8-analytics/fabric8-analytics-pgbouncer/commit/d0382139615300652cac21de034d1395cf787dc2

This change was done by app sre. any reason why you are updating it?

Reply:
Because build is failing even with that url.

Reply:
Last metadata expiration check: 0:00:01 ago on Tue Jul 27 09:04:03 2021.
[MIRROR] pgdg-redhat-repo-latest.noarch.rpm: Status code: 404 for https://download.postgresql.org/pub/repos/yum/10/redhat/rhel-8-x86_64/pgdg-redhat-repo-latest.noarch.rpm (IP: 147.75.85.69)
[MIRROR] pgdg-redhat-repo-latest.noarch.rpm: Status code: 404 for https://download.postgresql.org/pub/repos/yum/10/redhat/rhel-8-x86_64/pgdg-redhat-repo-latest.noarch.rpm (IP: 147.75.85.69)
[MIRROR] pgdg-redhat-repo-latest.noarch.rpm: Status code: 404 for https://download.postgresql.org/pub/repos/yum/10/redhat/rhel-8-x86_64/pgdg-redhat-repo-latest.noarch.rpm (IP: 147.75.85.69)
[MIRROR] pgdg-redhat-repo-latest.noarch.rpm: Status code: 404 for https://download.postgresql.org/pub/repos/yum/10/redhat/rhel-8-x86_64/pgdg-redhat-repo-latest.noarch.rpm (IP: 147.75.85.69)
[FAILED] pgdg-redhat-repo-latest.noarch.rpm: Status code: 404 for https://download.postgresql.org/pub/repos/yum/10/redhat/rhel-8-x86_64/pgdg-redhat-repo-latest.noarch.rpm (IP: 147.75.85.69)
Status code: 404 for https://download.postgresql.org/pub/repos/yum/10/redhat/rhel-8-x86_64/pgdg-redhat-repo-latest.noarch.rpm

Reply:
I think this should be the EL-8 repo, not EL-6

Reply:
ACK